app/main/views/register.py

```diff
@@ -174,5 +190,16 @@ def set_up_your_profile():
         # activate the user
         user = user_api_client.get_user_by_uuid_or_email(user_uuid, user_email)
         activate_user(user["id"])
-        return redirect(url_for("main.show_accounts_or_dashboard", next=redirect_url))
+        usr = User.from_id(user["id"])
+        usr.add_to_service(
+            invited_service.id,
+            invite_data["permissions"],
+            invite_data["folder_permissions"],
+            invite_data["from_user_id"],
+        )
+        current_app.logger.debug(
+            hilite(f"Added user {usr.email_address} to service {invited_service.name}")
+        )
+        return redirect(url_for("main.show_accounts_or_dashboard"))
```

Comment:
Do we still need the `redirect_url` piece or no?  I know it's removed from above, so just checking that this due to the change in flow as a consequence of the Login.gov integration.

Reply:
I don't know what the purpose of it is, so I removed it.  I wanted more confidence I would land on the "accounts or dashboard" page. :)